src/reynir_correct/config/GreynirCorrect.conf

```diff
@@ -3582,6 +3582,143 @@ sjálfstæðismaður
 Lenínismi
 Marxismi
 Darwinismi
+Heimsstyrjöld	# Fyrri/Seinni/Síðari Heimsstyrjöld
+Stríð 			# Kalda Stríðið
+Heimsveldi
+Bylting
+Uppreisn
+Meistaradeild
+Kreppa
+Endurreisn
+Jörð
+Tungl
+
+# Holidays
+# Unnecessary to include: föstudagurinn langi
+# Unable to include: annar í páskum, sumardagurinn fyrsti, baráttudagur verkalýðsins, annar í hvítasunnu, frídagur verslunarmanna, fyrsti vetrardagur, dag íslenskrar tungu, annar í jólum
+
+Nýársdagur
+Þrettándi
+Bóndadagur
+Konudagur
+Bolludagur
+Sprengidagur
+Öskudagur
+Páskar
+Pálmasunnudagur
+Skírdagur
+Páskadagur
+Mæðradagur
+Uppstigningardagur
+Hvítasunnudagur
+Hvítasunna
+Sjómannadagur
+Þjóðhátíðardagur
+Verslunarmannahelgi
+Gleðiganga
+Menningarnótt
+Hrekkjavaka
+Feðradagur
+Fullveldisdagur
+Jól
+Aðfangadagur
+Aðfangadagskvöld
+Jóladagur
+Áramót
+Gamlársdagur
+jónsmessa
+valentínusardagur
+þorláksmessa
+
+# Job titles
+# Unable to include: félags- og barnamálaráðherra, fjármála- og efnahagsráðherra, utanríkis- og þróunarsamvinnuráðherra, utanríkis- og þróunarsamvinnuráðherra, umhverfis- og auðlindaráðherra, sjávarútvegs- og landbúnaðarráðherra, mennta- og menningarmálaráðherra, samgöngu- og sveitarstjórnarráðherra, ferðamála-, iðnaðar- og nýsköpunarráðherra, sérstakur saksóknari, sr., hr.
+
+Ráðherra
+Dómsmálaráðherra
+Félagsmálaráðherra
+Barnamálaráðherra
+Fjármálaráðherra
+Efnahagsráðherra
+Utanríkisráðherra
+Umhverfisráðherra
+Forsætisráðherra
+Sjávarútvegsráðherra
+Landbúnaðarráðherra
+Menntamálaráðherra
+Menningarmálaráðherra
+Samgönguráðherra
+Sveitarstjórnarráðherra
+Heilbrigðisráðherra
+Forseti
+Saksóknari
+Ríkissaksóknari
+Aðalsaksóknari
+Konungur
+Kanslari
+Formaður
+Hreppstjóri
+Séra    
+Herra
+Frú
+Biskup
+Borgarstjóri
+Doktor
+Hæstaréttardómari
+Landlæknir
+Prestur
+Prófessor
+Sýslumaður
+Lögreglustjóri
+Ríkislögreglustjóri
+Alþingismaður
+Alþingiskona
+
+# Weekdays
+
+Mánudagur
+Þriðjudagur
+Miðvikudagur
+Fimmtudagur
+Föstudagur
+Laugardagur
+Sunnudagur
+
+# School subjects
+
+Eðlisfræði
+Efnafræði
+Félagsfræði
+Hagfræði
+Íþróttir
+Jarðfræði
+Kvikmyndafræði
+Landafræði
+Latína
+Líffræði
+Lýðheilsufræði
+Myndlist
+Næringarfræði
+Saga
```

Comment:
Be careful with this - is also a person name

Reply:
Ah of course! I replaced it with 'sagnfræði'